src/nimblepkg/tools.nim

```diff
@@ -98,6 +98,16 @@ proc copyFileD*(fro, to: string): string =
   copyFileWithPermissions(fro, to)
   result = to
 
+proc writeContents*(filename, contents: string) =
```

Comment:
You can achieve the same with just a ``try: writeFile(...) except OSError: raise(NimbleError, ... )`` so I don't think this proc is necessary.

Reply:
Okay. I'll change to use `writeFile` and delete it